src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,8 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(new Lottery().getRandomBall().toString());
+        }
```

Comment:
![image](https://user-images.githubusercontent.com/117547096/205385802-5ca9c7fe-0351-41a8-aa3b-df0b6fc88777.png)

```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
```